x/btcstaking/types/multi_staking_allow_list_placeholder.go

```diff
@@ -0,0 +1,19 @@
+//go:build !mainnet && !testnet
+
+package types
+
+// This is a placeholder for the multi-staking allow list and allow list expiration height.
+// Values here are not used in production and are only for testing purposes.
+// The actual allow list is defined in the files with build tags:
+// multi_staking_allow_list_mainnet.go and multi_staking_allow_list_testnet.go
+// Make sure to include the correct tx hashes for the
+// mainnet or testnet as needed in the corresponding
+// files with the build tags
+const multiStakingAllowListExpirationHeight = 5
```

Comment:
maybe we can leave default as `0` i.e disabled allow list ? this way it won't accidentally crash some e2e tests in repos depending on babylon.

Reply:
But while building the docker image, will pick up the mainnet values by default. Unless we change the flag for building the docker image with the desired values